libs/community/langchain_community/vectorstores/singlestoredb.py

```diff
@@ -424,13 +425,47 @@ def add_texts(
                             json.dumps(metadata),
                         ),
                     )
+                    cur.execute("SELECT LAST_INSERT_ID();")
+                    row = cur.fetchone()
+                    if row:
+                        ids.append(str(row[0]))
                 if self.use_vector_index or self.use_full_text_search:
                     cur.execute("OPTIMIZE TABLE {} FLUSH;".format(self.table_name))
             finally:
                 cur.close()
         finally:
             conn.close()
-        return []
+        return ids
```

Comment:
docstring needs to be updated

how do you feel about a `return_ids: bool = False` kwarg?

Reply:
Added